test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -102,14 +102,20 @@ void checkBlocks(TestBlock const& _blockFromFields, TestBlock const& _blockFromR
 
 void doBlockchainTests(json_spirit::mValue& _v, bool _fillin)
 {
-	if (!Options::get().fillchain) //fill blockchain through state tests
+	bool skipTestOutput = false;
+	string casename = boost::unit_test::framework::current_test_case().p_name;
+	if (casename == "bcRandom" || Options::get().fillchain)
```

Comment:
I don't like the test case names being hardcoded deep in the test framework code. Could we maybe have a dynamic option in `Options::get()` that has a list of test case names for which we skip output ?